cypress/e2e/zenscrape/documentation.cy.js

```diff
@@ -0,0 +1,112 @@
+describe('Zenscrape documentation page', () => {
+
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // Returning false here prevents Cypress from failing the test
+        return false;
+    });
+
+    before(() => {
+        cy.visit('https://zenscrape.com/');
+    });
+
+    it('1. Test to accept cookies if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to click the documentation hyperlink', () => {
+        cy.get('.gap-8 > [href="//app.zenscrape.com/documentation"]').click();
+    });
+
+    it('3. Test to check the text in the documentation section', () => {
+        cy.wait(8000);
+        //Added an 8-second wait to allow the documentation to load. A more reliable solution should be implemented to handle this wait.
+        cy.componentVisiblityCheck(':nth-child(1) > .col-md-12 > h4', 'Documentation');
+    });
+
+    it('4. Test to check if the "Register Free API Key" button is present', () => {
+        cy.componentVisiblityCheck('aside > .btn', 'Register Free Apikey');
+    });
+
+    it('5. Test to click the "Register Free API Key" button', () => {
+        cy.get('aside > .btn').click();
+        cy.navigateUrlwithCookies('https://app.zenscrape.com/documentation');
+    });
+
+    it('6. Test to check if the pro tip alert message is present', () => {
+        cy.componentVisiblityCheck('.alert', 'Pro Tip: Register your free apikey here and all code snippets below will contain your private apikey. If you have already registered, login before viewing the documentation.');
+    });
+
+    it('7. Test to check if the "Postman Collection" text and "Run in Postman" button are present', () => {
+        cy.componentVisiblityCheck('#postman', 'Postman Collection');
+        cy.componentVisiblityCheck('.postman-run-button');
+    });
+
+    xit('8. Test to check the "Run in Postman" button functionality', () => {
+        cy.get('.postman-run-button').click();
+        cy.wait(5000);
+        //Added an 5-second wait to allow the postman web page to load. A more reliable solution should be implemented to handle this wait.
+        cy.url().should('eq', 'https://app.getpostman.com/run-collection/85f8b936e3e1870c3fa9');
+        cy.go('back');
+    });
+
+    it('9. Test to navigate to the "Credit Costs & Failed Requests" section', () => {
+        cy.get('.mb-3 > :nth-child(1) > .nav-link').click();
```

Comment:
what if the hierarchy changes ?

Reply:
There is no way to get the element other than selector and the selector is having the class like this only.